src/main/resources/db/migration/all/20240627100000__update_temporary_accommodation_assessments.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE temporary_accommodation_assessments
+    ADD COLUMN release_date DATE NULL;
+ALTER TABLE temporary_accommodation_assessments
+    ADD COLUMN accommodation_required_from_date DATE NULL;
```

Comment:
@froddd just tagging you in here as this change may affect your work on editing release dates